src/pages/components/button/usage.mdx

```diff
@@ -186,6 +188,8 @@ The button is available in seven different sizes: **extra small**, **small**,
 For guidance on button heights, see [Size](/components/button/style/#size) on
 the Style tab.
 
+Carbon does not recommend mixing different button sizes in #button-groups.
```

Comment:
Do we normally refer to ourselves as "Carbon"? Just curious, because I've never seen it before. Maybe something like "We do not recommend..." or "Mixing button sizes is not recommended..."?

Reply:
The last two options listed are more correct. It's fine to use 'We' in the documentation. 'Carbon' sounds a little awkward here.